First case.R

```diff
@@ -0,0 +1,92 @@
+library(tidyverse)
+library(lubridate)
+
+#Calculates start date for each country
+
+#col_num <- apply(cases[,5:ncol(cases)], 1, function(x){first_case = min(which(x>0))})
+
+#first_case <- names(cases[,5:ncol(cases)])[col_num]
+
+#cases <- cbind(first_case, cases)
+
+
+# reading in files
+
+cases = read.csv(".\\intermediate_data\\jhu_cases_01.26.2021.csv", check.names = FALSE)
+
+deaths = read.csv(".\\intermediate_data\\jhu_deaths_01.26.2021.csv", check.names = FALSE)
+
+population = read.csv(".\\intermediate_data\\world_bank_population.csv", check.names = FALSE)
+
+indicators <- read.csv(".\\intermediate_data\\ghsi_summary.csv", check.names = FALSE)
+
+
+#pivoting dates into rows and grouping by country code and date
+
+cases_country_date <- cases[7:376]
+
+cases_pivot <- pivot_longer(cases_country_date, cols= !contains("country_code"), names_to = "Date", values_to = "Cases")
+
+cases_pivot_grouped <- aggregate(cases_pivot$Cases, by = list(Category = cases_pivot$country_code, cases_pivot$Date), FUN=sum)
+
+deaths_country_date <- deaths[7:376]
+
+deaths_pivot <- pivot_longer(deaths_country_date, cols= !contains("country_code"), names_to = "Date", values_to = "Deaths")
+
+deaths_pivot_grouped <- aggregate(deaths_pivot$Deaths, by = list(Category = deaths_pivot$country_code, deaths_pivot$Date), FUN=sum)
+
+
+#renaming columns
+
+cases_pivot_grouped <- cases_pivot_grouped %>% 
+  rename(
+    country_code = Category,
+    Date = Group.2,
+    Cases = x
+  )
+
+deaths_pivot_grouped <- deaths_pivot_grouped %>% 
+  rename(
+    country_code = Category,
+    Date = Group.2, 
+    Deaths = x
+  )
+
+#merging deaths and cases
+
+deaths_and_cases <- merge(cases_pivot_grouped, deaths_pivot_grouped, by = c("country_code"="country_code", "Date"="Date"))
+
+
+# nonzero cases matrix, 
+
+nonzero_cases <- deaths_and_cases[deaths_and_cases$Cases >0, ]
```

Comment:
Inside a data table you don't need to refer to the name of the data table again. This can just be deaths_and_cases[Cases>0]. Not a big deal to change though.